ClangSharp.PInvokeGenerator/Cursors/Exprs/Expr.cs

```diff
@@ -138,14 +138,25 @@ public static new Expr Create(CXCursor handle, Cursor parent)
             }
         }
 
+        private readonly Lazy<Cursor> _definition;
         private readonly Lazy<Type> _type;
 
         protected Expr(CXCursor handle, Cursor parent) : base(handle, parent)
```

Comment:
In the C++ type hierarchy, `Expr` is actually a sub-type of `Stmt`. I'll be looking at how to expose that later.